### ISSUE TITLE
All Pages - 2.4.1 Bypass Blocks - A tabindex is present on the Skip To Content link

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9905]
As a user dependant on the keyboard and screen reader to navigate the page, I need to be able to rely on bypass links to take me to the correct place in the page and to ensure correct tabbing order. 

**Linked Work Item:**
#7978  

**Expected Result:**
Tabindexes should not be present on Skip To links as they indicate a specific order for page navigation (this is especially important to remove if you want to remove heading levels entirely).

**Actual Result:**
The presence of a tabindex on the Skip To Content link has the potential to modify the tabbing order and therefore cause confusion for users, so should be avoided where possible.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/de70b4a2-6e03-4418-8dc1-9a95ccb2d16b)
